modules/social_features/social_event/modules/social_event_invite/src/SocialEventInviteAccessHelper.php

```diff
@@ -96,8 +96,18 @@ public function eventFeatureAccess() {
     }
 
     // Get the group of this node.
-    $node = $this->routeMatch->getRawParameter('node');
-    $node = Node::load($node);
+    $node = $this->routeMatch->getParameter('node');
+    if (!$node instanceof NodeInterface) {
+      // On views url parameters node can be provided as raw parameter.
+      $nid = $this->routeMatch->getRawParameter('node');
+      // Node can't be loaded.
+      if (empty($nid)) {
+        return AccessResult::neutral();
```

Comment:
Is this idea different from : https://github.com/goalgorilla/open_social/pull/3084/files#diff-19022dfe4a0998c73d0a6e27f8c20fd5c748ba1885cb3471b9e08e44e3aef3bcR42

Reply:
@nechai The idea of returning AccessResults is a bit different from what I linked above. So, I wanted to know why there are differences in logic?

Reply:
@navneet0693 Now I got what you mean - yes, definitely should be replaced with  `AccessResult::forbidden()` (but even current code will not break anything)

Reply:
@nechai You are right, this is not breaking anything :-D but I would to request to also place the same awesome idea logic here as well. The idea is really a good one.

Reply:
@navneet0693 done